### PR TITLE
Yelp reviews

### PR DESCRIPTION
**Pull request by abirkus**
adding google reviews is impossible without paying for places API. 
I found an easy solution to embed iframe for google location and yelp reviews.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -110,6 +123,87 @@ const Home: NextPage = () => {
           </Box>
         </Box>
       </BgImage>
+      <Box
```
Comment (edfarrera) [inline src/pages/index.tsx:126]: This looks like a lot of content. Can we move summarize it in another component?

File: src/components/yelp/yelpReviews.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { Box } from '@mui/material';
+import React from 'react';
+import { useEffect } from 'react';
+
+const reviewsIds = [
```
Comment (edfarrera) [inline src/components/yelp/yelpReviews.tsx:5]: Do we want/need this ids hard coded?

File: src/components/yelp/yelpReviews.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <div key={reviewId} style={{ padding: '15px', minWidth: '400px' }}>
+      <span
+        className="yelp-review"
+        data-review-id={reviewId}
+        data-hostname="www.yelp.com"
+      ></span>
+    </div>
+  ));
+  useEffect(() => {
+    const scriptTag = document.createElement('script');
```
Comment (edfarrera) [inline src/components/yelp/yelpReviews.tsx:30]: Do we need to do this via a script? Is there a more react way to do it?

File: src/pages/placeorder.tsx [hunk 4]
```diff
@@ -119,19 +119,26 @@ function PlaceOrder() {
             </Typography>
             <TableContainer>
               <Table>
-                {summaryArr.map((el) =>
-                  el[1] ? (
-                    el[0] === 'dropoffDate' || el[0] === 'pickupDate' ? (
+                {formEntriesKeyValueArray.map((formFieldKeyValueArr) =>
```
Comment (edfarrera) [inline src/pages/placeorder.tsx:122]: This whole file is quite big already, we need to see which elements can be extracted to get better code readability and control

File: src/components/yelp/yelpReviews.tsx (hunk 2 above)
Comment (abirkus) [reply src/components/yelp/yelpReviews.tsx:5]: we only want to pull the 5 star reviews. we can use an API call to fetch the data but then we would need to clean it up before each render. this is simple solution that should work for now.

File: src/components/yelp/yelpReviews.tsx (hunk 3 above)
Comment (abirkus) [reply src/components/yelp/yelpReviews.tsx:30]: that I am not sure about. I tried to look up Yelp api, and in order to get access to it, you need to register with yelp and they need to approve your access. 

There is a way to embed reviews by copying their HTML link which included the script to their widgets.js to customize and style the tags. without the script, reviews dont look good. maybe there is a better way to do this.

File: src/pages/index.tsx (hunk 1 above)
Comment (abirkus) [reply src/pages/index.tsx:126]: good point, will do.

File: src/pages/placeorder.tsx (hunk 4 above)
Comment (abirkus) [reply src/pages/placeorder.tsx:122]: feel free to refactor. i wasn't sure how to make it better.

**Pull request merged**